system/jlib/jsecrets.cpp

```diff
@@ -804,3 +804,64 @@ IPropertyTree *queryMtlsSecretInfo(const char *name)
     }
     return info;
 }
+
+enum UseMTLS { UNINIT, DISABLED, ENABLED };
+static UseMTLS useMtls = UNINIT;
+
+jlib_decl bool queryMtls()
+{
+    if (useMtls == UNINIT)
+    {
+        useMtls = DISABLED;
+#if defined(_USE_OPENSSL)
+        const char *mtlsName = "mtls";
```

Comment:
I use the name "local" for the local cluster tls certificates.  If you change to that name it should work with the generated cert-manager certificates.

Reply:
@afishbeck - The feature name in the proptree and the path to the cert seem to me to be one in the same ?
I don't see how we can, with the code as is, have a feaure name in the proptree of "mtls" and from that use a path of "local" ?

Reply:
ok, I changed things and use "local" for the proptree for the cert path and "mtls" for the feature.